docs/General/release-process.md

```diff
@@ -0,0 +1,111 @@
+# Release process
+
+Creating a new release involves creating a new release of the design
+system, the react components and the CMS itself. It's important that
+it's done in that order, due to the dependencies between the projects.
+
+In the following you'll need to use the "Bypass branch protection"
+option when merging to skip having someone review the changes.
+
+# Design system
+https://github.com/danskernesdigitalebibliotek/dpl-design-system
+
+1. Create a pull request from the `develop` branch against `main`.
+2. Wait for Github Actions to run and turn green.
+3. Merge the pull request.
+4. [Create a new
+release](https://github.com/danskernesdigitalebibliotek/dpl-design-system/releases/new)
+     1. Enter the release version as "Choose a tag", and select to
+        create it on publish.
+     2. Select `main` as target.
+     3. Click "Generate release notes" to automatically fill out title
+        and description.
+     4. Ensure that "Set as the latest release" is selected before
+        publishing the release.
+
+
+# Update DPL React for local development
+
+These steps are not critical for release, but are required for
+developers to get the latest version of the design system when
+developing.
+
+You might not have the required permissions for this step, in that
+case, continue to releasing DPL React and return to this later.
```

Comment:
Consider adding a link to https://danskernesdigitalebibliotek.github.io/dpl-docs/DPL-React/#requirements which describes how to authenticate against the GitHub NPM repository.

Currently the wording is not totally clear here. You might read it specific GitHub permissions might be needed or the like but it is really something anyone working with DPL React would be able to do.